src/modules/person/persistence/person.repository.ts

```diff
@@ -118,6 +185,12 @@ export class PersonRepository {
         return mapEntityToAggregateInplace(personEntity, personWithKeycloakUser);
     }
 
+    public async delete(person: Person<true>): Promise<number> {
```

Comment:
can this be removed or at least set private? It should not be possible to delete without permissions.

Reply:
Removed, not necessary.